sidecar/src/interface.rs

```diff
@@ -682,7 +726,9 @@ impl SidecarServer {
             None => return,
         };
 
-        session.shutdown().await
+        info!("Shutting down session: {}", session_id);
+        session.shutdown().await;
+        debug!("Successfully shut down session: {}", session_id);
```

Comment:
It seems like maybe the first item should be `debug!` and the second should be `info!`?

Reply:
The intention was noting the moment in time where no new data is accepted for the session. The second is really just "successfully cleaned up".